cachito/web/static/api_v1.yaml

```diff
@@ -277,6 +277,33 @@ paths:
                 $ref: "#/components/schemas/RequestConfiguration"
       security:
       - "Kerberos Authentication": []
+  "/requests/{id}/content-manifest":
+    get:
+      description: Return the content manifest of a request
+      parameters:
+      - name: id
+        in: path
+        required: true
+        description: The ID of the Cachito request to retrieve the content manifest for
+        schema:
+          type: integer
+      responses:
+        "200":
+          description: The content manifest of the Cachito request
+          content:
+            application/json:
+              schema:
+                $ref: "https://raw.githubusercontent.com/containerbuildsystem/atomic-reactor/f4abcfdaf8247a6b074f94fa84f3846f82d781c6/atomic_reactor/schemas/content_manifest.json#"
```

Comment:
Not sure how to get this one to work. It seems it is not happy with `"type": ["object", "null"]` and it just shows null. The file is being loaded, since the schema title is being imported.

Reply:
I'm not sure JSON schema is supported as OpenAPI is an extended subset of JSON schema:
https://swagger.io/docs/specification/data-models/keywords/

Reply:
Any suggestions to avoid duplicating the schema here?

Reply:
I ended up partially pulling in the schema